rpc/lib/server/handlers.go

```diff
@@ -532,20 +535,21 @@ func (wsc *wsConnection) writeRoutine() {
 		case <-wsc.Quit:
 			return
 		case <-wsc.pingTicker.C:
+			wsc.baseConn.SetWriteDeadline(time.Now().Add(time.Second * wsWriteTimeoutSeconds))
```

Comment:
Ahhh... this fixed the issue?  Maybe a short comment to link to the issue, so we know why Deadline is important in 4 months.